docs/content/preview/deploy/multi-dc/async-replication/async-transactional-failover.md

```diff
@@ -0,0 +1,391 @@
+
+1. Resume the application traffic on the new Primary (B).
+
+  </div>
+
+</div>
+
+After completing the preceding steps, the former Standby (B) is the new Primary (active) universe. There is no Standby until the former Primary (A) comes back up and is restored to the correct state.
+
+## Set up reverse replication
+
+If the former Primary universe (A) doesn't come back and you end up creating a new cluster in place of A, follow the [steps for a fresh replication setup](../async-transactional-setup/).
+
+If universe A is brought back, to bring A into sync with B and set up replication in the opposite direction (B->A), the database on A needs to be dropped and recreated from a backup of B (Bootstrap).
+
+<ul class="nav nav-tabs-alt nav-tabs-yb custom-tabs">
+  <li>
+    <a href="#localreverse" class="nav-link active" id="local-tab" data-toggle="tab"
+      role="tab" aria-controls="local" aria-selected="true">
+      <img src="/icons/database.svg" alt="Server Icon">
+      Local
+    </a>
+  </li>
+  <li>
+    <a href="#anywherereverse" class="nav-link" id="anywhere-tab" data-toggle="tab"
+      role="tab" aria-controls="anywhere" aria-selected="false">
+      <img src="/icons/server.svg" alt="Server Icon">
+      YugabyteDB Anywhere
+    </a>
+  </li>
+</ul>
+<div class="tab-content">
+  <div id="localreverse" class="tab-pane fade show active" role="tabpanel" aria-labelledby="local-tab">
+
+Do the following:
+
+1. Disable PITR on A as follows:
+
+    - List the snapshot schedules to obtain the schedule ID:
+
+        ```sh
+        ./bin/yb-admin -master_addresses <A_master_addresses> \
+        list_snapshot_schedules
+        ```
+
+    - Delete the snapshot schedule:
+
+        ```sh
+        ./bin/yb-admin \
+            -master_addresses <A_master_addresses> \
+            delete_snapshot_schedule <schedule_id>
+        ```
+
+1. Drop the database(s) on A.
+
+    If the original cluster A went down and could not clean up any of its replication streams, then dropping the database may fail with the following error:
+
+    ```output
+    ERROR: Table: 00004000000030008000000000004037 is under replication
+    ```
+
+    If this happens, do the following using yb-admin to clean up replication streams on A:
+
+    - List the CDC streams on Cluster A to obtain the list of stream IDs as follows:
+
+        ```sh
+        yb-admin \
+        -master_addresses <A_master_ips> \
+        -certs_dir_name <dir_name> \
+        list_cdc_streams
+        ```
+
+    - For each stream ID, delete the corresponding CDC stream:
+
+        ```sh
+        yb-admin \
+        -master_addresses <A_master_ips> \
+        -certs_dir_name <dir_name> \
+        delete_cdc_stream <streamID>
+        ```
+
+1. Recreate the database(s) on A.
+
+    If you are using YubabyteDB Anywhere, don't create any tables. Replication setup (Bootstrapping) will create tables and objects on A from B.
```

Comment:
You are mentioning about YBA in a local setup ? I think you can delete line 331.